### PR TITLE
stars displaying proper number now

### PR DESCRIPTION
**Pull request by ellenkramp**
changed functions to get star amount and create route card so that proper number of stars are displayed

### CODE REVIEW

File: javascript/detail_and_listview.js [hunk 1]
```diff
@@ -226,15 +245,9 @@ var makeDetailView = function(id) {
         userImage.src = snapshot.val()['thumbnail'];
         var startLocation = snapshot.val()['startLocation'];
         var pois = snapshot.val()['pois'];
-        if (walkerLocation !== null){
```
Comment (ponchieponcho) [inline javascript/detail_and_listview.js]: just double checking to see if these changes need to stay or that you removed them on purpose

Comment (ellenkramp) [reply javascript/detail_and_listview.js]: good point... I feel like I might be overriding someone else's changes... Let me double check

Comment (rfpoulos) [reply javascript/detail_and_listview.js]: good catch, we definately need to account for behavior if users don't volunteer their location.

File: javascript/detail_and_listview.js [hunk 2]
```diff
@@ -75,8 +92,7 @@ var getAmountOfStars = function(div) {
     div.appendChild(imgReviewStars4);
     div.appendChild(imgReviewStars5);
 };
-
-var createRouteCardSkeleton = function(object) {
+var createRouteCardSkeleton = function(object, id) {
```
Comment (rfpoulos) [inline javascript/detail_and_listview.js]: why do we need two arguments?  what is the difference between object and id?

**Pull request closed without merging**